[website] Add designer position page (#30708)
mui-org/material-ui: docs/src/pages/company/careers/designer.md

# Designer

<p class="description">Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.</p>

## About the company

MUI started as Material-UI, the most successful React implementation of Google’s Material Design.
It has gotten a massive following not only due to the fidelity to Material but also because its number of components, carefully designed component API, obsession for details, and community engagement.
Today, countless teams and organizations rely on our open-source projects to build their design system.

A few years back, we started expanding our suite of products.

> We released [MUI X](/x/), a collection of advanced components, [MUI Design Kits](/design-kits/), the MUI Material components available in the most popular design tools out there, and [MUI Templates](/templates/), a set of pre-built UIs.

Reviewer: This is not factual, MUI Design kits is meant to include all the React components, Joy, X, Material. 

```suggestion
We released [MUI X](/x/), a collection of advanced components, [MUI Design kits](/design-kits/), the MUI components available in the most popular design tools out there, and [MUI Templates](/templates/), a set of pre-built UI.
```

We've seen fast growth in all of them (x2 ~ 3 year over year) and we have much more baking in the oven.

We are a team of fully distributed individuals, spread across multiple timezones.
We work mainly asynchronously, relying on lots of documentation and written collaboration.
Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
For more details about MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).

## Why we're hiring

Material-UI started as an open-source project.
As years passed by, audience and usage grew considerably, even turning some of the library's original founders turned full-time maintainers of it.
A few years ago, the project was incorporated and just very recently, [we have rebranded the company from Material-UI to MUI](/blog/material-ui-is-now-mui/).

> A new brand was fundamental to set the stage for MUI to become larger than Material Design, which was something we actually were already doing.

Reviewer: Simpler?

```suggestion
A new brand was fundamental to set the stage for MUI to become larger than Material Design, which was something we were already doing.
```

Right now, we have five main projects that are not at all related to MD:

- [MUI Core:](/core/)
  - [MUI Base:](/customization/unstyled-components/) the unstyled version of the components.
  - [MUI System:](/system/basics/) our most recent take on styling for React components.
  - [MUI Joy (working title)](https://github.com/mui-org/material-ui/discussions/29024): a second design system, alternative to Material Design.
- [MUI X:](/x/) as mentioned already, a growing set of advanced components.
Today, the flagship one is the [Data Grid](/components/data-grid/), a game-changing component for dealing with lots of data, which integrates perfectly with MUI Core.
- MUI Studio: a very recent endeavor aimed at enabling users to visually create apps 10x faster with the power of low-code and the flexibility of pro-code.

We also learned, especially due to [our yearly run developer surveys](/blog/2020-developer-survey-results/), that, for developers, design quality plays a huge part when considering which available component library to use.
Therefore, we need to grow the design organization to help us push it further.

## About the role

> Some criterias for applying to this role:

Reviewer: ```suggestion
Some criteria for applying to this role:
```

- Location: fully remote (preference for UTC-6 to UTC +3).

> - Type of work: full-time (as a contractor or employee, [depending primarily on location](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501)).

Reviewer: @mbrookes We were using this wording before:

```suggestion
- Type of work: full-time (as a contractor or employee, [depending on the case](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501)).
```

Are you sure about the change? With Sam, we have https://www.letsdeel.com/ set up now. The page on Notion says:

> In specific occurrences, MUI can use an Employer of Record (EOR) (we use letsdeel.com/) to resolve specific issues but we haven't generalized this practice.

Reviewer 2: > In specific occurrences, MUI can use an Employer of Record (EOR) (we use letsdeel.com/) to resolve specific issues but we haven't generalized this practice.

Hence it's primarily based on location (employee for France, contractor otherwise, occasional exception).

Reviewer: @mbrookes OK, so this would be an accurate description of what we have done in the past. I agree.

However will it keep describing what we do in the future? The main problem I see is: are we missing out on better talents because the description? If we are, then I don't think that it's worth it. Are we? I don't know, but keeping it open ended could help us learn this.

Reviewer 2: "depending on circumstances" will fix the grammar without changing the current meaning.

Reviewer: Alright. I'm updating this, and we can move this discussion to a follow-up, this will unlock this PR to be merged.
